### PR TITLE
Updating Miniconda links

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #13571 

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

**Review by kenodegard — APPROVED** [review]

**Review by kenodegard — CHANGES_REQUESTED** [review]
theres actually a few more instances of the old miniconda URL scattered throughout the code (not just in the docs), e.g.:
- https://github.com/conda/conda/blob/99902ee6914bead092f78842a81c5439758dd006/README.md#L32
- https://github.com/conda/conda/blob/99902ee6914bead092f78842a81c5439758dd006/HOW_WE_USE_GITHUB.md#L228 (to be updated within https://github.com/conda/infrastructure)

**Comment by travishathaway** [comment]
@kenodegard,

Thanks for catching that. I will make the necessary updates.

**Review by kenodegard — CHANGES_REQUESTED** [review]
Some more:
- https://github.com/conda/conda/blob/12f301c5b7621464c8aaecdc6d913c6ba68b78d5/docs/source/glossary.rst?plain=1#L251
- https://github.com/conda/conda/blob/12f301c5b7621464c8aaecdc6d913c6ba68b78d5/docs/source/user-guide/install/linux.rst?plain=1#L7
- https://github.com/conda/conda/blob/12f301c5b7621464c8aaecdc6d913c6ba68b78d5/docs/source/user-guide/troubleshooting.rst?plain=1#L406

**Comment by travishathaway** [comment]
@kenodegard,

Thanks for catching that. Ready for another look.

**Review by kenodegard — DISMISSED** [review]

File: news/13572-conda-sphinx-docs-version-0-2-1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* Bump version of conda-sphinx-docs to 0.2.1 (#13572)
```
Comment (beeankha) [inline news/13572-conda-sphinx-docs-version-0-2-1:15]:
```suggestion
* Update the navigation links for Miniconda. (#13572)
```

**Review by beeankha — DISMISSED** [review]
Other than the news file change suggestion, this LGTM!

**Review by kenodegard — APPROVED** [review]

**Pull request merged**